packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -2,12 +2,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import withStyles from '@material-ui/core/styles/withStyles';
+import { fade, withStyles } from '@material-ui/core/styles';
 import ButtonBase from '@material-ui/core/ButtonBase';
-import { fade } from '@material-ui/core/styles/colorManipulator';
-import { setRef } from '@material-ui/core/utils/reactHelpers';
-import withForwardedRef from '@material-ui/core/utils/withForwardedRef';
-import clamp from '../utils/clamp';
+import { setRef, withForwardedRef } from '@material-ui/core/utils';
+import { clamp } from '@material-ui/lab/utils';
```

Comment:
At some point we should consolidate all the utils:
- `@material-ui/core/utils`
- `@material-ui/lab/utils`
into `@material-ui/utils`